src/Page/Food/Builder.elm

```diff
@@ -771,8 +784,8 @@ sidebarView session db model results =
         ]
 
 
-stepListView : Db -> Model -> Recipe -> Recipe.Results -> Html Msg
-stepListView db { impact, selectedPackaging, selectedTransform } recipe results =
+stepListView : Db -> List Country -> Model -> Recipe -> Recipe.Results -> Html Msg
+stepListView db countries { impact, selectedPackaging, selectedTransform } recipe results =
```

Comment:
Same remark as to putting countries in the Db. Note that I put these remarks everywhere to simplify looking after the issue in the codebase and ease later reviews ;)